### ISSUE TITLE
[Bug] No internet access in anbox

### ISSUE THREAD

**Issue by chillenzer** [ubports/ubuntu-touch#2140]
- Device: OnePlus ONEPLUS A6013
- Channel: 16.04 stable
- Build: OTA-25

Hi,

first of all: Thanks for the great work! I use Ubuntu Touch in production for a few years now and while it has significantly more hickups that my friend's phones, it's still the one I feel most comfortable with. 

That being said, it got almost unusable for me a while ago when internet access in anbox broke down.

### Steps to reproduce
```
$ adb shell
$ ping 80.241.218.68
```
(or try to use any android apps that want internet access)

### Expected behavior
```
$ adb shell
$ ping 80.241.218.68
PING 80.241.218.68 (80.241.218.68) 56(84) bytes of data.
64 bytes from 80.241.218.68: icmp_seq=1 ttl=242 time=51.9 ms
...
```

### Actual behavior
```
$ adb shell
$ ping 80.241.218.68
connect: Network is unreachable
```

### Logfiles and additional information
The closest I've got to a solution by now is [this issue](https://github.com/anbox/anbox/issues/443) in anbox. For me, the shown routes and rules look (almost) identical
```
$ adb shell
$ ip route
192.168.250.0/24 dev eth0  proto kernel  scope link  src 192.168.250.2
$ ip rule
0:	from all lookup local 
10000:	from all fwmark 0xc0000/0xd0000 lookup 99 
13000:	from all fwmark 0x10063/0x1ffff lookup 97 
13000:	from all fwmark 0x10064/0x1ffff lookup 1036 
15000:	from all fwmark 0x0/0x10000 lookup 99 
16000:	from all fwmark 0x0/0x10000 lookup 98 
17000:	from all fwmark 0x0/0x10000 lookup 97
```
But the described workarounds fail because apparently the anbox version shipping with Ubuntu Touch doesn't have `su(do)` installed?
```
$ adb shell
$ su
/system/bin/sh: su: not found
```
(Neither does any of the other things work that you find on the web like anbox-bridge or the like.)

But [this comment](https://github.com/anbox/anbox/issues/443#issuecomment-531455247) made me think. Wasn't there something about IPv6 [in another issue](https://github.com/ubports/ubuntu-touch/issues/1657) lately? Could this (or rather the related commits that might or might not have made it into my phone) cause the issue?

In summary: If possible, with some pointers to how to get root access into my anbox, I'd likely be able to solve the problem myself. Otherwise, could we find a better way to have both, mobile data _and_ internet access in anbox?

**Comment by Danfro** [comment]
There is no on working on Anbox anymore. Anbox is more or less dead. Also development on Xenial has been stopped too, focal is being worked on. So sadly do not expect any fixes here.

Maybe Waydroid might work for you. That is actively developed.

**Comment by chillenzer** [comment]
Okay. Thanks for the clarification. I'm happily using Waydroid on my laptop already. Is there a supported (or at least suggested) way to get it working on my device? AFAIK, the latter is not yet supported by the new Ubuntu Touch where that is the default.

**Comment by Danfro** [comment]
I am afraid I don't know much about Waydroid except the fact that it does exist. Best ask the porter of your device if Waydroid can be enabled.
